### PR TITLE
docs: add job name to workflow example

### PR DESCRIPTION
**Pull request by wyardley**
Without the "job" having a name, it doesn't seem to be possible to add this workflow as a required check in GH branch protection.

Add this to the example in the README

### CODE REVIEW

**Comment by amannn** [comment]
Hi and thank you for your PR!

Did you see this error somewhere?

In this repo I'm not using a name currently:

https://github.com/amannn/action-semantic-pull-request/blob/4feeedbc065232b05e657d69d072355d0c7cb8ea/.github/workflows/lint-pr-title-preview.yml#L9-L21

**Comment by wyardley** [comment]
> Did you see this error somewhere?

> In this repo I'm not using a name currently:

Hi! So, it doesn't create an error, and does properly show up in both the status checks on the main PR page and in the "checks" / "actions" pages, however, without the `name` parameter on the job itself, the check doesn't seem to be able to become a "required" status check for GitHub branch protection, which it doesn't look like you're using. It was extremely non-obvious, and there weren't that many posts out there that had the correct fix, but stumbled upon
https://stackoverflow.com/questions/68554735/github-action-status-check-missing-from-the-list-of-checks-in-protected-branch-s

I do think it's probably a common use case in many types of environment to have certain status checks "required" in branch protection settings, and I couldn't find a way to have this check enabled without adding the name parameter to the job itself. (in the UI, you kind of have to type ahead to find the checks, so the fact that they're missing is super confusing too). Screenshot:

image

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ on:
 
 jobs:
   main:
+    name: check_pr_title
```
Comment (amannn) [inline README.md:33]: What do you think about using something human readable like it is suggested in the [GH actions docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-5)?

```suggestion
    name: Validate PR title
```

**Review by amannn — COMMENTED** [review]
Thank you for your detailed explanation! I agree, this is really subtle and it's probably helpful if we support this use case better.

File: README.md (hunk 1 above)
Comment (wyardley) [reply README.md]: Sure - I rebased off of current trunk, and updated the name to match your suggestion.

**Comment by amannn** [comment]
Thanks a lot! I'm sure this will be helpful to others 🙂

**Pull request merged**